dev/maven-plugin/src/main/resources/META-INF/maven/plugin.xml

```diff
@@ -0,0 +1,384 @@
+<!-- TODO - this should be at least in part generated -->
+
+<!-- This was copied from https://maven.apache.org/ref/3.3.9/maven-plugin-api/plugin.html.
+  Unused configuration items have simply been commented out in case they need to be used
+  in future -->
+<plugin>
+    <name>Lagom Maven Plugin</name>
+    <description>Provides maven support for Lagom Framework</description>
+    <groupId>com.lightbend.lagom</groupId>
+    <artifactId>lagom-maven-plugin</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
```

Comment:
I guess this should be 1.0.1-SNAPSHOT - unless you plan on generating this file before merging.